internal/log_analysis/rules_engine/src/engine.py

```diff
@@ -51,13 +51,17 @@ def analyze_single_rule(self, raw_rule: dict, test_spec: Mapping) -> Dict[str, A
         raw_rule['versionId'] = 'default'
         rule = Rule(raw_rule)
         event = test_spec['data']
+        if 'mocks' not in test_spec:
+            event_mocks = dict()
+        else:
+            event_mocks = test_spec['mocks']
```

Comment:
Could this be condensed

```suggestion
        event_mocks = test_spec.get('mocks', dict()):
```